### ISSUE TITLE
Accessibility: No name is defined for button

### ISSUE THREAD

**Issue by formulahendry** [microsoft/vscode#212323]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.89.0
- OS Version: Windows 11 Enterprise 23H2 (OS build 22631.3296)

Copied from https://github.com/microsoft/vscode-azureapicenter/issues/100

### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG4.1.2; #GH_VSCodeextensionforAPICenter_Win32_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Name Role Value; #AILimited; #A11yMAS;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:

1. Create a API center in azure portal.
2. Open Visual studio Code.
3. Tab till Chat option in the left side panel and press enter.
4. Type @workspace /tests in the copilot chat box.
5. Verify whether the name is defined for "How can I write unit tests for python function" button or not.

### Actual Result:
No name is defined for "How can I write unit tests for python function" button.

### Expected Result:
Name should be defined for "How can I write unit tests for python function" button.

### User Impact:
Screen reader users will not get the information about the functionality of the button if name is not defined.

### Attachment: 
![Bug_100](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/b4167511-c7f3-4555-98d2-c2674c90ce35)
